journal/updated/01-journal.adoc

```diff
@@ -767,251 +767,270 @@ nevertheless, afterward it yields the peaceable fruit of righteousness,
+The apostle tells us, "`That the law came by Moses,
 but grace and truth came by Jesus Christ.`"
 I further told them,
 "`that holy men of God gave them forth as they were moved by the Holy Spirit.`"
 As to this, they seemed to be satisfied.
 
-But the priest put some ensnaring questions to me; and when I perceived it,
-I asked the justices what that man was; whether he was a justice of the peace or not;
-and whether he was not a priest; they said,
-they looked upon him to be a fitter man than
+But the priest began to ask me some ensnaring questions, and when I perceived it,
+I asked the justices who this man was,
+whether he was a justice of the peace or not,
+or whether he was only a priest. They said
+they considered him to be a fitter man than
 themselves to discourse with me about religion.
-I told them, I thought he was as the high-priest among the Jews,
-who put ensnaring questions to Christ, when he was brought before them,
-to seek to make him an offender; and turned myself to the justices,
-and desired them to take notice of that man, who laid those ensnaring questions,
+I told them I thought he was as the high-priest among the Jews,
+who asked Christ ensnaring questions when He was brought before them,
+to seek to make Him an offender. And turning to the justices,
+I desired them to take notice of that man who asked those ensnaring questions
 to seek to make me an offender.
 
 Then the priest left me, and the justices asked me, seeing I would not take the oaths,
-whether I would give bail; and said,
+whether I would give bail; and said
 they would take my father`'s bail for my good behaviour.
-I told them, my cause was just, and I was innocent, and would give them no bail,
-for truth binds me to my good behaviour.
-Then the high-sheriff, a very fair man, told me, I was a strange man,
+I told them, "`My cause is just, and I am innocent, and will give you no bail,
+for truth binds me to my good behaviour.`"
+Then the high-sheriff, a very fair man, told me I was a strange man,
 and of a strange persuasion, to come with my hat upon my head among them,
-and would not take the oaths nor give bail.
-You know, said he, that Paul said to Festus, Noble Festus.
-I told him, that Paul had tried Festus, but I had not as yet tried him; and it might be,
-that I might speak of him, Noble Sheriff, upon this they were most of them very pleasant.
-He asked me, whom I did know there; there were several justices my relations present,
-who very well knew me, but I made mention of none of them; but told them,
+and not take the oaths nor give bail.
+"`You know,`" said he, "`that Paul said to Festus, '`Noble Festus.`'`"
+I told him that Paul had tried Festus, but I had not yet tried him;
+and it may be, that after I had tried him, I might say to him "`Noble Sheriff;`"
+and upon saying this most of them seemed very pleasant.
+He asked me whom I knew there; there were several justices present
```

Comment:
His probably should be a complete sentence. He asked me **who** I knew there. There were several...